torchbox/torchbox.com
# Make related_sectors optional and fix works property on WorkIndexPage

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1398716868)

### Description of Changes Made

This ensures that Work Pages, Historical Work Pages, Blog Pages and Events can be created / updated without specifying the sectors taxonomies.

While testing the changes, I noticed that the filtered QuerySet had duplicate results, so I fixed this by using `distinct()` on the QuerySet.

### How to Test

- Add a couple of Sector Taxonomies
- Edit or create a Work Page, Historical Work Page, Blog Page or Event
  - you should be able to save it with or without specifying a sector taxonomy
- Check that the filtering on the Work Index Page works as expected

### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot from 2024-02-19 09-51-35](https://github.com/torchbox/torchbox.com/assets/7713776/6d0afd61-c5ea-42bf-a8d2-743bdb3ee4b3)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on Linux
  - Latest version of Firefox on Linux

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
